### DISCUSSION TITLE
Federation forced logout

### DISCUSSION THREAD

**Discussion by iabetor** [minio/minio#20048] [Q&A] [answered]
I used federation expansion to add a new cluster. However, when I log into the console of the first cluster, the account on the console of the second cluster gets logged out. Is this normal behavior?

**Comment by harshavardhana** [comment, marked as answer]
There isn't anything in the product called federation expansion,

Federation was supported a long time ago where two distinct deployments can stay together in some loose sense.

So not sure what you are doing here and you haven't shared much details either for us to help you.

**Reply by iabetor** [reply]
After using federation, I now have two clusters. I logged into the console of one cluster using its IP and port, but when I log into the console of the second cluster, the first console gets logged out. Is this normal behavior? I hope I’ve explained it clearly.

**Reply by harshavardhana** [reply]
Yes of course the login credentials are valid only for a single domain/IP.  You need to have a common endpoint for both clusters for your logins to work both ways.

**Reply by iabetor** [reply]
thank you
